repo/store.go

```diff
@@ -25,6 +26,69 @@ type Refstore interface {
 	RefCount() (int, error)
 }
 
+// GetVersionInfoShim is a shim for getting away from the old
+// DatasetRef, CanonicalizeDatasetRef, RefStore stack
+func GetVersionInfoShim(r Repo, ref dsref.Ref) (*dsref.VersionInfo, error) {
```

Comment:
Are these really shims or will this be permanent here?

Reply:
Long term, I'm hoping we can remove these packages entirely, I think the `Shim` language should stay so others see a hint to avoid using these APIs